test/albums/graphql.js

```diff
@@ -0,0 +1,35 @@
+const { gql } = require('apollo-server');
+
+const getAlbum = id => gql`
+  query {
+    album(id: ${id}) {
+      id
+      title
+      artist
+      photos {
+        id, title
+      }
+    }
+  }
+`;
+const getAlbums = params => {
+  const { offset, limit, orderBy, filter } = params;
+  return gql`
+  query {
+    albums(
+      offset: ${offset ? offset : 0}, 
+      limit: ${limit ? limit : 0}, 
+      orderBy: "${orderBy ? orderBy : ''}",
```

Comment:
is this field an enum? you might want to create an enum type instead of keeping it as strings

Reply:
I thought about it when I did that sorting part, but I ended up not using it because I thought it wasn't worth it for an application this small. But I'll do it now.